Activate some Driver-Options and change the DefaultInputSlot from Auto Feeder to Cassette.

--- bjfilter-2.50/ppd/canonpixusip4100.ppd        2005-01-14 06:50:44.000000000 +0100
+++ bjfilter-2.50/ppd/canonpixusip4100.ppd        2006-03-29 16:56:13.000000000 +0200
@@ -55,6 +55,14 @@
 *ParamCustomPageSize Orientation: 5 int 1 1
 *CustomPageSize True: "pop pop pop <</PageSize [5 -2 roll] /ImagingBBox null>>setpagedevice"

+*OpenUI *CNQuality/Quality: PickOne
+*DefaultCNQuality: 3
+*CNQuality 2/High: "2"
+*CNQuality 3/Normal: "3"
+*CNQuality 4/Standard: "4"
+*CNQuality 5/Economy: "5"
+*CloseUI: *CNQuality
+
 *OpenUI *PageSize/Paper Size: PickOne
 *DefaultPageSize: a4
 *PageSize a5/A5: "<</CNPageSizeName(a5)/PageSize[420 595]/ImagingBBox null>>setpagedevice"
@@ -108,7 +116,7 @@
 *CloseUI: *MediaType

 *OpenUI *InputSlot/Paper Feed: PickOne
-*DefaultInputSlot: asf
+*DefaultInputSlot: cassette
 *InputSlot asf/Auto Feeder: ""
 *InputSlot cassette/Cassette: ""
 *CloseUI: *InputSlot
@@ -116,6 +124,8 @@
 *OpenUI *Resolution/Output Resolution: PickOne
 *DefaultResolution: 600
 *Resolution 600/600 dpi: "<</HWResolution[600 600]>>setpagedevice"
+*Resolution 1200/1200 dpi: "<</HWResolution[1200 1200]>>setpagedevice"
+*Resolution 2400/2400 dpi: "<</HWResolution[2400 2400]>>setpagedevice"
 *CloseUI: *Resolution

 *OpenUI *ColorModel/Color Model: PickOne
